features/admin.server-configurations.v1/forms/username-recovery-form.tsx

```diff
@@ -16,45 +16,26 @@
  * under the License.
  */
 
-import { IdentifiableComponentInterface } from "@wso2is/core/models";
+import { AppConstants, history } from "@wso2is/admin.core.v1";
+import {
+    UsernameRecoveryFormConstants
+} from "@wso2is/admin.server-configurations.v1/constants/username-recovery-constants";
+import {
+    UsernamePasswordRecoveryFormUpdatableConfigsInterface,
+    UsernameRecoveryConfigurationFormPropsInterface,
+    UsernameRecoveryFormValuesInterface
+} from "@wso2is/admin.server-configurations.v1/models/username-recovery";
+import { CommonUtils } from "@wso2is/core/utils";
 import { Field, Form } from "@wso2is/form";
+import { Heading, Link, Message } from "@wso2is/react-components";
 import isEmpty from "lodash-es/isEmpty";
 import React, { FunctionComponent, ReactElement, useEffect, useState } from "react";
+import { Trans, useTranslation } from "react-i18next";
+import { Icon } from "semantic-ui-react";
```

Comment:
Can we use the oxygen icon here?

Reply:
Fixed by https://github.com/wso2/identity-apps/pull/7004/commits/4e7d6afe4c4846d96cca9cf71746852a1809ef69

Updated the message component to oxygen ui alert component.